### PR TITLE
Typo correction in FAQ and Corrected misinformation

### PR DESCRIPTION
**Pull request by hyeyeon919**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

While translating the app in Korean I was able to find a typo error in the FAQ.
I corrected storgea to storage.
Next is the location where the file is saved is currently changeable.
However, the said non-changeable. So I deleted the word. non-changealbe.

Thank you.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by codegsaini — CHANGES_REQUESTED** [review]
Commit title is out of way from your proposed changes :(

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -578,7 +579,7 @@
...
-        <item>How to change the default storgae location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and click the <strong>Select icon</strong> at the top. \nOnce changed, the files created using the app will be stored in the new location.</item>
+        <item>How to change the default storage location?#####To change the default storage location, \n1) Click the <strong>Hamburger icon</strong> (three horizontal line) at the top-left corner. \n2) Scroll down and click <strong>Settings</strong>. \n3) In the settings page, click the field just beneath the text <strong>Change storage location</strong>. \n4) Open the preferred save location and click the <strong>Select icon</strong> at the top. \nOnce changed, the files created using the app will be stored in the new location.</item>
```
Comment (codegsaini) [inline app/src/main/res/values/strings.xml:582]: Good job, please mention this changes in pull request's description.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -469,7 +469,7 @@
     <string name="font_family_value_def">Font family default value : %s </string>
     <string name="settings_info">You can modify the default value for various settings here.</string>
     <string name="storage_location_modified">Storage location changed</string>
-    <string name="storage_location_info">Storage location(non-changeable)</string>
+    <string name="storage_location_info">Storage location</string>
```
Comment (codegsaini) [inline app/src/main/res/values/strings.xml:472]: Good

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -568,6 +568,7 @@
     <string name="snackbar_files_deleted">Files Deleted</string>
     <string name="lbl_recently_used">Recently Used Features</string>
 
+
```
Comment (codegsaini) [inline app/src/main/res/values/strings.xml:571]: Unnecessary change :(

File: app/src/main/res/values-ko-rKR/arrays.xml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
+
```
Comment (codegsaini) [inline app/src/main/res/values-ko-rKR/arrays.xml]: Unnecessary change :(

File: app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java [hunk 5]
```diff
@@ -1,8 +1,6 @@
 package swati4star.createpdf;
 
 import android.content.Context;
-import android.support.test.InstrumentationRegistry;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
```
Comment (codegsaini) [inline app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java:5]: Changes are different than what title is telling about.

**Review by codegsaini — CHANGES_REQUESTED** [review]
Please add proper description in pull request description about proposed changes and also do some changes mentioned with individual comment.

File: app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java (hunk 5 above)
Comment (hyeyeon919) [reply app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java:5]: Thank you for letting me know about the problems. I'm sorry I am not a good user of GitHub :(
I closed my PRs add oped new ones. And I fixed everything you said. 
I hope you check it out soon :)
Thank you for your feedback!

**Comment by hyeyeon919** [comment]
Hi 
I fixed the problem you commented.
Please check my new PRs.
And it would be nice if you merge it !!
Thank You! :)
-----Original Message-----
